### ISSUE TITLE
Docutils version

### ISSUE THREAD

**Issue by hugobowne** [coiled/feedback#80]
A bug report from [twitter](https://twitter.com/michaelaye/status/1311119528064155649) (thanks @michaelaye!):

> Why does coiled insist on docutils 0.15.2 and uninstalls my conda docutils 0.16? Is it possible to remove that restrictive requirement? It breaks conda environments like this.

![image](https://user-images.githubusercontent.com/4182382/94638440-56323d00-031d-11eb-8690-c39f4c1bf1e4.png)

**Comment by jrbourbeau** [comment]
Thank @michaelaye for raising this point (and @hugobowne for opening up an issue here). Today, `coiled`s dependencies are:

```
aiohttp
aiobotocore>=1.0.7
click
dask[complete]>=2.23.0
distributed>=2.23.0
s3fs
pandas>=1.1.0
ipython
```

I'm not immediately sure where the `docutils` version constraint is getting pulled in from. I recall `aiobotocore` having a somewhat restrictive set of dependencies, so maybe that's a good place to start. 

Since you mentioned using conda environments, you may also consider installing `coiled` from conda-forge

```
conda install -c conda-forge coiled
```

**Comment by michaelaye** [comment]
It's `aiobotocore` requirement on `botocore` which, in its conda build history, has the weird property of sometimes depending on `docutils<0.16` and sometimes not. The required version of botocore 1.17.44 is resolved to a build that does require docstring<0.16.

**Comment by FabioRosado** [comment]
Hello, I'm going through the open issues on this repository and close some of them. As of today, our version of coiled (0.0.35) uses botocore 1.20.2, which might fix the issue that you were experiencing. I've also tried to install all those dependencies on a new conda environment, and the docutils wasn't downgraded.

I'm closing this issue now, but please feel free to re-open or open a new issue if you experience any further problems
